src/sql/workbench/contrib/charts/browser/graphInsight.ts

```diff
@@ -266,13 +275,19 @@ export class Graph implements IInsight {
 				}
 			}
 
-			retval.legend = <Chart.ChartLegendOptions>{
-				position: options.legendPosition as Chart.PositionType,
-				display: options.legendPosition !== LegendPosition.None,
-				labels: {
-					fontColor: foreground
+			retval.plugins = {
+				legend: {
+					position: options.legendPosition as any,
```

Comment:
Why any? You should be making the actual position type here, otherwise we risk this silently breaking later if either of the types changes.